src/main/scala/strawman/collection/mutable/SortedSet.scala

```diff
@@ -1,12 +1,17 @@
 package strawman
 package collection.mutable
 
+import scala.Ordering
+
+import strawman.collection.ConstrainedIterablePolyTransforms
+
 trait SortedSet[A]
   extends collection.SortedSet[A]
     with Set[A]
     with SortedSetLike[A, SortedSet]
 
 trait SortedSetLike[A, +C[X] <: SortedSet[X]]
   extends collection.SortedSetLike[A, C]
+    with ConstrainedIterablePolyTransforms[A, Set, SortedSet]
```

Comment:
By the way, would it be possible to pull up this line at the level of `collection.SortedSet` so that we don’t repeat it for `collection.mutable.SortedSet` and `collection.immutable.SortedSet`?